qt/python/mantidqtinterfaces/mantidqtinterfaces/dns_powder_tof/file_selector/file_selector_presenter.py

```diff
@@ -0,0 +1,281 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+"""
+DNS File selector Presenter - Tab of DNS Reduction GUI.
+"""
+
+from mantidqtinterfaces.dns_powder_tof.data_structures.dns_observer import \
+    DNSObserver
+
+
+class DNSFileSelectorPresenter(DNSObserver):
+    def __init__(self,
+                 name=None,
+                 parent=None,
+                 view=None,
+                 model=None,
+                 watcher=None):
+        # pylint: disable=too-many-arguments
+        super().__init__(parent=parent, name=name, view=view, model=model)
+        self.watcher = watcher
+        self.view = view
+        # set sample data view
+        self.view.set_tree_model(self.model.get_model())
+        # set standard data view
+        self.view.set_tree_model(self.model.get_model(standard=True),
```

Comment:
It would be better to have view have two functions, one to set the sample tree and one to set the standard tree

Reply:
Right, thanks for the suggestion! Will fix that